packages/batteries/batteries.2.5.0/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "1.2"
+name: "batteries"
+maintainer: "thelema314@gmail.com"
+authors: "OCaml batteries-included team"
+homepage: "http://batteries.forge.ocamlcore.org/"
+bug-reports: "https://github.com/ocaml-batteries-team/batteries-included/issues"
+dev-repo: "https://github.com/ocaml-batteries-team/batteries-included.git"
+license: "LGPL-2.1+ with OCaml linking exception"
+doc: "http://ocaml-batteries-team.github.io/batteries-included/hdoc2/"
+
+build: [
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  [make "all"]
+]
+install: [
+  [make "install"]
+]
+remove: [["ocamlfind" "remove" "batteries"]]
+
+depends: [
+  "ocamlfind" {>= "1.5.3"}
```

Comment:
It's not clear to me that `build` would be enough, given that for <4.02 versions we rely on the `Bytes` compatibility package it provides, which should also be linked by user programs.